tests/runners/test_helpers.py

```diff
@@ -73,14 +73,14 @@ def test_bad_licenses_identified(self):
         tree = get_tree()
 
         whitelist = ["MIT", "Apache-2.0"]
-        _, unknown_licenses = annotate_dep_tree(tree, whitelist)
+        _, unknown_licenses = annotate_dep_tree(tree, whitelist, [])
```

Comment:
This is hardly a test for this functionality

Reply:
This is just to make this test backwards compatible. I'll add a test for this functionality in a different test case. 👍